bin/circleci/run-integration-tests.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
+NODE_ENV=production ENABLE_PONTOON=1 ENABLE_DEV_CONTENT=1 ENABLE_DEV_LOCALES=1 npm run static
```

Comment:
Don't run this here - run it in [`bin/build-frontend.sh`](https://github.com/mozilla/testpilot/blob/master/bin/circleci/build-frontend.sh) by removing the conditional so this runs on all branches and not just master.

Reply:
You can basically copy & paste what I have [over in this comment from the previous PR](https://github.com/mozilla/testpilot/pull/2828#issuecomment-326405418) into `bin/circleci/build-frontend.sh`

Reply:
Well I would like this to work locally, if a user wants to run tests against the site. What do you think?

Reply:
You can run the same `npm run static` command locally before running this. That, or `npm start` like we usually do for local development.

Reply:
I also just realized that this `run-integration-tests` script doesn't actually run integration tests. It just starts a server.

But it doesn't stop an existing server or stop it after a test run. And it gets run [in tox.ini](https://github.com/mozilla/testpilot/pull/2842/files#diff-b91f3d5bd63fcd17221b267e851608e8R11) for every test run. That probably means there's an error starting up the server a 2nd or 3rd time since the port's already taken.

We could probably move this script out of tox.ini and just run it once in circle.yml before the test-firefox-*.sh scripts - and maybe rename it to `start-server.sh` or something

Reply:
Will the server be accessible even throughout different subprocesses? With the problems we had with circle in the beginning, it is nice to keep everything within the same shell process, as it seems they are pretty aggressive when it comes to killing things running in different shells.

Reply:
It's in a subprocess now - that's what the `&` in `npm run server &` does